Makefile

```diff
@@ -61,12 +61,15 @@ watch-unit:
 watch-compile:
 	$(BIN_DIR)/tsc --watch  --noEmit --skipLibCheck
 
+oathkeeper:
+	oathkeeper serve -c dev/ory/oathkeeper.yml --sqa-opt-out
+
 e2e:
+	# need oathkeeper running in another terminal
```

Comment:
if it is need it why not execute the cmd here?

Reply:
it's a server so it doesn't return. it has to stay on until the test complete.

it would be possible to do a `authkeeper & yarn build && yarn test:e2e` but if authkeeper error you would not see it.

any suggestion how to do better?